deepchem/feat/base_classes.py

```diff
@@ -75,6 +76,60 @@ def _featurize(self, datapoint: Any):
     """
     raise NotImplementedError('Featurizer is not defined.')
 
+  def __repr__(self) -> str:
```

Comment:
> We should add __repr__ methods to all our featurizers :)

I think we don't need to do this. This is because these implementation is put on the `Featurizer` class.

So, this implementation is valid for all featurizers except featurizers inherited ComplexFeaturizer

Reply:
Ah you're right!